Installer/configuration.json

```diff
@@ -0,0 +1,32 @@
+{
+    "SQLServer": {
```

Comment:
We should add a parameter to select if we want to install to MSSQL or Postgres. With this version we cannot install the views if we don't have both databases.